packages/components/image.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { connect, Head } from "frontity";
+import { Connect, Package } from "frontity/types";
+import useInView from "@frontity/hooks/use-in-view";
+
+// Hides any image rendered by this component that is not
+// inside a <noscript> when JS is disabled.
+const noJsStyles = `
+  :not(noscript) > .frontity-lazy-image {
+    display: none;
+  }
+`;
+
+// Finds all the images rendered by this component and maps
+// their `data-src` and `data-srcset` attributes to `src` and `srcset`
+// when the browser doesn't support Proxy or IntersectionObserver.
+const noProxyScript = `
+  if (typeof window !== "undefined" && (!("Proxy" in window) || !("IntersectionObserver" in window))) {
+    document.addEventListener("DOMContentLoaded", () => {
+      const images = document.querySelectorAll("img.frontity-lazy-image");
+      for (image of images) {
```

Comment:
This script needs to be in es5 but I'll do the fix.

Reply:
Maybe we could use `raw-loader` in the future to transpile and minify this kind of scripts? Is it possible? I don't remember from the previous version.

Reply:
Hmm.. I'm not sure if that's possible because `raw` is... well, raw 😆 

My idea a while back was to give packages the possibility of adding an additional entry-point that loads before React for this type of things.